src/main/java/com/notably/logic/commands/NewCommand.java

```diff
@@ -29,10 +30,13 @@ public NewCommand(Block block, AbsolutePath path) {
      */
     public void execute(Model notablyModel) throws CommandException {
         requireNonNull(notablyModel);
-
-        if (notablyModel.hasPath(this.path)) {
-            throw new CommandException("Block with the same Title detected.");
+        try {
+            if (notablyModel.hasPath(this.path)) {
+                throw new CommandException("Block with the same Title detected.");
+            }
```

Comment:
We can actually remove this portion right, if my understanding is correct?

Reply:
Yes it was redundant.